Shared/zh-Hans.lproj/Localizable.strings

```diff
@@ -138,6 +138,15 @@
 
 "SAVE_TO_PHOTOS" = "保存到照片";
 
+// History
+"HISTORY" = "历史";
+"%@_AGO" = "%@ ago"; // x days ago
```

Comment:
```suggestion
"%@_AGO" = "%@前"; // x days ago
```
Is keeping the string untranslated intentional? In usage, `%@` will be replaced with a proper phrase for your language so you don't need to worry about things not making sense.